documentation/book/oauth/proc-oauth-client-config.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following module:
+//
+// con-oauth-config.adoc
+
+[id='proc-oauth-client-config-{context}']
+= Configuring Kafka Java clients to use {oauth}
+
+This procedure describes how to configure Kafka producer and consumer APIs to use {oauth} for interaction with Kafka brokers.
+
+Add a client callback plugin to your _pom.xml_ file, and configure the system properties.
+
+.Prerequisites
+
+* AMQ Streams and Kafka is running
+* {oauth} authorization server is deployed and configured for OAuth access to Kafka brokers
+* Kafka brokers are configured for {oauth}
+
+.Procedure
+
+. Add the client callback plugin to the `pom.xml` file for the Kafka client:
++
+[source,xml,subs="+attributes"]
+----
+<dependency>
+ <groupId>io.strimzi</groupId>
+ <artifactId>kafka-oauth-client</artifactId>
+ <version>{oauth-artifact-version}</version>
+</dependency>
+----
+
+. Configure the system properties for the callback:
++
+For example:
++
+[source,xml, subs="+quotes,attributes"]
+----
+System.setProperty(ClientConfig.OAUTH_TOKEN_ENDPOINT_URI, “_https://<authorization-server-address>/auth/realms/demo/protocol/openid-connect/token_”); <1>
+System.setProperty(Config.OAUTH_CLIENT_ID, "_<client-name>_"); <2>
+System.setProperty(Config.OAUTH_CLIENT_SECRET, "_<client-secret>_"); <3>
```

Comment:
I think we can use `ClientConfig` for both of these to make the code a bit nicer (@mstruk The `ClientConfig` extends `Config`, right?)

Reply:
Right. It would indeed make it look nicer.

Reply:
Updated